src/coap_mbedtls.c

```diff
@@ -300,6 +300,25 @@ get_error_string(int ret) {
   return buf;
 }
 
+static int
+self_signed_cert_verify_callback_mbedtls(void *data,
+                                         mbedtls_x509_crt *crt COAP_UNUSED,
+                                         int depth COAP_UNUSED,
+                                         uint32_t *flags)
+{
+  coap_session_t *c_session = (coap_session_t*)data;
```

Comment:
You could make most variables here `const` to document that no change is intended.

Reply:
Code updated and pushed